### DISCUSSION TITLE
Clarification regarding differential privacy guarantee offered by β

### DISCUSSION THREAD

**Discussion by jaintj95** [OpenMined/courses#171] [Course: Our Privacy Opportunity] [answered]
Lesson 4: Towards Robust Output Privacy Infrastructure
Timestamp - 16:00 

Andrew mentions a medical study which guarantees no more than 2 β of information will be released.
Does 2 β imply 2 times β or β = 2?

Because if β is 2, then wouldn't the total loss be 2 * β = 2 * 2 = 4?

**Comment by adtygan** [comment]
It means no more than β value of 2 will be released. 
Now, β = e^ε - (1)
Taking β = 2, Andrew mentions that the ε value won't exceed roughly 0.7
Plug in 0.7 to the equation (1) and you will see β comes out to be 2.01

**Comment by mmalekzadeh** [comment]
I agree with @AdityaG09's response. 

One more thing in this Lesson which I could not completely understand was when Aron Roath mentions something like "e to the epsilon times one half": here at this point https://youtu.be/XwtCeYdzTXc?t=829

Does he mean something like  Pr(presence) < 1/2e^{\epsilon} Pr(absence) ? 
Does anyone know where this "times one half" comes from? Is that because the random guess is 1/2? 
I might be missing something.

**Comment by adtygan** [comment]
I had the same doubt @mmalekzadeh . Can someone please explain this one?

**Comment by tanmaylaud** [comment]
@mmalekzadeh , @AdityaG09 , yes the 0.5 is because of the guess being random ( just a baseline condition, like an unbiased coin toss). In general multiplying e^eps by the Probability of an event will give you the maximum possible information leakage given that event. (Based on my understanding)

**Comment by leriomaggio** [comment, marked as answer]
hi @jaintj95 
   are you satisfied with the answers you received ? 
if there is still anything which is not clear enough, or want to understand better, please feel free to let us know :) We are very happy to help! 🙌 
When you feel you're happy with the support, it would be great if you could mark this question as answered ✅ 

Thank you so much!
